plugins/modules/aix_filesystem.py

```diff
@@ -66,7 +66,7 @@
     type: str
   filesystem:
     description:
-      - Specifies the mount point, which is the directory where the file system will be mounted.
+      - Specifies the mount point, which is the directory where the file system is being mounted.
```

Comment:
I think 'will be' is better here, since this modifies /etc/filesystem, and doesn't always mount the filesystem right away.

Reply:
done